src/h5web/visualizations/Visualizations.test.tsx

```diff
@@ -1,24 +1,21 @@
-import { fireEvent, screen } from '@testing-library/react';
-import { renderApp } from '../test-utils';
+import { screen } from '@testing-library/react';
+import { renderApp, selectExplorerNode } from '../test-utils';
 
 describe('Visualizer', () => {
-  test('visualise scalar dataset', async () => {
+  test('visualise raw dataset', async () => {
     renderApp();
+    await selectExplorerNode('entities/raw');
 
-    fireEvent.click(await screen.findByRole('treeitem', { name: 'entities' }));
-    const datasetBtn = screen.getByRole('treeitem', { name: 'scalar_int' });
-    expect(datasetBtn).toBeVisible();
+    expect(await screen.findByText(/"int": 42/u)).toBeVisible();
```

Comment:
Would it be possible to have a more robust way to get the value of the dataset ?

Reply:
I could do `JSON.stringify(mockValues.raw)` but it would be re-implementing the raw vis 😅 -- you can see it as a snapshot test. If it fails even though we didn't change the mock data, then there's a problem.